Graphite/GraphiteClient.cs

```diff
@@ -162,8 +163,9 @@ public Task<GraphiteMetricData[]> GetMetricsDataAsync(SeriesListBase target, str
         /// <param name="template">The target metrics can use a special <see cref="SeriesListBase.Template(string[])"/> function which allows the metric paths to contain variables</param>
         /// <param name="maxDataPoints"></param>
         /// <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="CancellationToken.None"/>.</param>
+        /// <param name="tz">specify time zone for from/until. Corresponds to 'tz' parameter in Graphite Render API.</param>
         /// <returns></returns>
-        public async Task<GraphiteMetricData[]> GetMetricsDataAsync(SeriesListBase[] targets, string from = null, string until = null, IDictionary<string,string> template = null, ulong? maxDataPoints = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<GraphiteMetricData[]> GetMetricsDataAsync(SeriesListBase[] targets, string from = null, string until = null, IDictionary<string,string> template = null, ulong? maxDataPoints = null, CancellationToken cancellationToken = default(CancellationToken), string tz = null)

```

Comment:
can we make the cancellationToken the last parameter?

Reply:
Yes but it will break existing usages. If you're OK with it I'll do it.

Reply:
fixed :)